cmd/mirror.go

```diff
@@ -221,80 +118,192 @@ func newMirrorSetCmd() *cobra.Command {
 	return cmd
 }
 
+// the `mirror introduce` sub command
+func newMirrorIntroduceCmd() *cobra.Command {
+	name := ""
+	privPath := ""
+
+	cmd := &cobra.Command{
+		Use:   "introduce <id>",
```

Comment:
How about using` tiup mirror grant/revoke` to add a new owner and remove exists owner.

Reply:
I don't think remove an owner is possible. The owner doesn't has a 'yank' field so we can just remove it from the manifest file. After that, the existed components owned by that owner can't be verified (since the key is removed with that owner) and there will be signature error when user execute `tiup list` or execute the installed component.

Reply:
Maybe we should ignore the components missing owner, and report warning when user already install that component.

Reply:
But what if something depends on it? 

IMO, delete owner directly breaks the `yank` mechanism